include/antidote.hrl

```diff
@@ -157,9 +157,12 @@
 
 -define(CLOCKSI_TIMEOUT, 1000).
 
+-type txn_properties() :: [{update_clock, boolean()} | {certify, use_default | certify | dont_certify}].
+
```

Comment:
I would leave this for the moment being and refactor the properties at a later time. I don't understand fully yet how the txn_properties work in combination with the default parameter.